src/js/components/tablero/Tablero.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Instead of this disabler, please add a role="button" on a div that has the onClick.